dotnet/samples/Concepts/Memory/VectorStore_MigrateFromMemoryStore_Redis.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+#if DISABLED
```

Comment:
@westey-m I've converted some other samples which show how to connect via MEVD to an IMemoryStore-created collection, but this one is slightly different since it shows an actual data migration... We should decide what we want to do with it (but can probably do it later?).